app/frontend/src/components/admin/ServiceClients.vue

```diff
@@ -0,0 +1,241 @@
+<template>
+  <v-card>
+    <v-toolbar flat color="grey lighten-3">
+      <v-card-title>Service Clients</v-card-title>
+    </v-toolbar>
+    <v-container>
+      <!-- search input -->
+      <div class="kc-search">
+        <v-text-field
+          v-model="search"
+          append-icon="mdi-magnify"
+          label="Search"
+          single-line
+          hide-details
+          class="pb-5"
+        ></v-text-field>
+      </div>
+      <!-- table alert -->
+      <v-alert v-if="showAlert" :type="alertType" tile dense>{{alertMessage}}</v-alert>
+      <!-- table header -->
+      <v-data-table
+        class="kc-table"
+        :headers="headers"
+        :items="serviceClients"
+        :items-per-page="10"
+        :search="search"
+        :loading="loading"
+        loading-text="Loading... Please wait"
+        :expanded.sync="expanded"
+        item-key="acronym"
+        show-expand
+
+      >
+        <!-- environments -->
+        <template v-slot:item.dev="{item}">
+          <div v-if="item.dev">
+            <v-tooltip top>
```

Comment:
promotion date shows in tooltip. would be nice to have this tool tip appear on click event as well for mobile users but i dont think that's working

Reply:
You can probably add some toggle logic by binding visibility to a v-model. Caveat is when doing this, I suspect the default hover event will no longer be listened to, so you'd have to re-implement that behavior in addition to the on-click toggle.
Reference: https://vuetifyjs.com/en/components/tooltips/#visibility